helm-org.el

```diff
@@ -137,6 +147,16 @@ NOTE: This will be slow on large org buffers."
           :buffer "*helm org inbuffer*")))
 
 ;;;###autoload
+(defun helm-org-parent-headings ()
+  "Preconfigured helm for org headings that are parents of the current heading."
+  (interactive)
+  (helm :sources (helm-source-org-headings-for-files
+                  (list (current-buffer))
+                  1 8 t)
```

Comment:
Sorry, I'm confused: if `(parent t)` were the default, then `helm-source-org-headings-for-files` would always show only parent headings, so all the other places that source is used would have to specify `(parents nil)` to avoid that.

However, I think it doesn't matter, because I was thinking about it last night, and I think the best thing to do is to specify a large `max-depth` anyway.  That way, if a user has more than 8 levels of headings (and I sometimes do), all of the parent headings will still be displayed.

I'll push another commit with the changes soon.  Thanks.